input/pagecontent/reference.md

```diff
@@ -1653,6 +1650,70 @@ A filter is a logical statement in the form `{property} {operator} {value}`, whe
 
 > **Note:** Intensional and extensional forms can be used together in a single value set definition.
 
+{%include tu-div.html%}
+
+##### Concept Metadata
+
+Within a ValueSet definition, the caret syntax MAY be used to set metadata attributes for individual concepts (e.g., elements of ValueSet.compose.include.concept.designation).
```

Comment:
MAY is a formal descriptor attached to a requirement for FSH language. This is a difference sense of "may".  I would say this as "the caret syntax can be used to"

Reply:
This mirrors a similar statement that is in the section on [Code System Code Metadata](https://hl7.org/fhir/uv/shorthand/reference.html#code-metadata) in FSH 2.0. (I copy/pasted and modified). Should I change it in both places?

Reply:
Yes, please change it. The formal "SHALL" "MUST" "SHOULD" terms are only appropriate when making statements about the FSH language.

Reply:
Done in d48596aa772190b3c223caa1883b4e974f3f970f.